ironmon_tracker/data/PokemonData.lua

```diff
@@ -158,6 +158,11 @@ function PokemonData.isValid(pokemonID)
 	return pokemonID ~= nil and pokemonID >= 1 and pokemonID <= PokemonData.totalPokemon
 end
 
+function PokemonData.isImageIDValid(pokemonID)
+	--Eggs (412), Ghosts (413), and placeholder (0)
+	return PokemonData.isValid(pokemonID) or pokemonID == 412 or pokemonID == 413
```

Comment:
Egg and Ghost image are valid, even though the pokemon are not. Also includes all normall valid pokemon

Reply:
Minor thing here, comment references an ID of 0 (implied it's acceptable) but the conditions do not account for this.

Is it true that in all cases this new function is called that it properly handles a pokemonID=0?